docs/en/administration/fault_diagnosis_and_analysis.md

```diff
@@ -11,15 +11,15 @@ import TabItem from '@theme/TabItem';
 
 ## Client log
 
-JuiceFS client will output logs for troubleshooting during operation, the log levels are from low to high: DEBUG, INFO, WARNING, ERROR, FATAL, by default only logs above INFO level are output. If you need to output DEBUG level logs, you need to explicitly enable it when running the JuiceFS client, such as adding the `--debug` option.
+JuiceFS client will output logs for troubleshooting during operation. Log levels in order of fatality are: DEBUG, INFO, WARNING, ERROR and FATAL. Since DEBUG logs are not printed by default, you need to explicitly enable it if you need, e.g. by adding the `--debug` option, when running the JuiceFS client, 
```

Comment:
```suggestion
JuiceFS client will output logs for troubleshooting while running. The level of logs in terms of fatality follows DEBUG < INFO < WARNING < ERROR < FATAL. Since DEBUG logs are not printed by default, you need to explicitly enable it if needed, e.g. by adding the `--debug` option when running the JuiceFS client. 
```
- Maybe "while running" is better than "during operation"?